velox/exec/Spill.h

```diff
@@ -172,7 +179,10 @@ class FileSpillBatchStream : public BatchStream {
   }
 
   bool nextBatch(RowVectorPtr& batch) override {
-    return spillFile_->nextBatch(batch);
+    if (spillFile_->nextBatch(batch)) {
```

Comment:
why make the change? It is the same as
```
return spillFile_->nextBatch(batch);
```

Reply:
I make a change before, dropped close() in the final version. I think we don't need to close the `FileSpillBatchStream`. Back it.